pages/common/od.md

```diff
@@ -0,0 +1,25 @@
+# od
+
+> Display file contents in hexadecimal, octal, or printable character format.
+> Optionally display the byte offsets into the file.
+
+- Display file in octal format 8 bytes per line with the byte offsets in octal as well replacing duplicate lines with '*':
+
+`od {{/path/to/file}}`
+
+- Display file in hexadecimal format with byte offsets in hexadecimal, 4 bytes per line with each entry 1 byte long:
+
+`od --address-radix=x --format=x1 --width=4 -v {{/path/to/file}}`
```

Comment:
forces not to skip duplicate lines in the output, if not set will output instead * (asterisk)
which in hex dumping a file is never a desired outcome.